src/control/tools/EditSelectionContents.cpp

```diff
@@ -523,13 +522,7 @@ void EditSelectionContents::serialize(ObjectOutputStream& out) {
 void EditSelectionContents::readSerialized(ObjectInputStream& in) {
     in.readObject("EditSelectionContents");
 
-    double originalWidth = in.readDouble();
-    double originalHeight = in.readDouble();
-
-    double originalX = in.readDouble();
-    double originalY = in.readDouble();
-
-    this->originalBounds = Rectangle<double>{originalX, originalY, originalWidth, originalHeight};
+    this->originalBounds = Rectangle<double>{in.readDouble(), in.readDouble(), in.readDouble(), in.readDouble()};
```

Comment:
This is less readible; side effect order should be explicit.

Reply:
Ok, I will change it back then.